packages/ui/v2/core/Shell.tsx

```diff
@@ -153,12 +153,12 @@ const Layout = (props: LayoutProps) => {
 
       {/* todo: only run this if timezone is different */}
       <TimezoneChangeDialog />
-      <div className="h-screen overflow-hidden">
-        <div className="flex h-screen overflow-hidden" data-testid="dashboard-shell">
+      <div className="flex min-h-screen flex-col">
```

Comment:
This is actually the main change, moving the banners outside the page container and on top of everything. This did however cascade into a lot of other changes that needed to be made in our Shell to even be able to acommodate this  😅 — those are the changes following next.